index.md

```diff
@@ -7,6 +7,15 @@ layout: default
 OpenAstronomy is a collaboration between open source astronomy and astrophysics
 projects to share resources, ideas, and to improve code.
 
+## Principles of OpenAstronomy
+
+The core principles of OpenAstronomy (OA) include:
+
+1. Open Source: OA projects are committed to their software being Open Source[1]. This promotes accessibility, reliability, and scientific reproducibility.
+[1] https://opensource.org/docs/osd
```

Comment:
move references to the bottom, and make numbering continual (e.g. not two [1], but a [1] and a [2]